README.adoc

```diff
@@ -24,6 +24,11 @@ toc::[]
 [[img-build-status]]
 image::https://ci.centos.org/buildStatus/icon?job=vagrant-service-manager[link="https://ci.centos.org/job/vagrant-service-manager"]
 
+== Prerequisites
```

Comment:
Is this really needed? By the time you deal with the vagrant-service-manager you already have a VM setup, right? Nothing against being explicit, but this feels too much too me.